src/redux/configureStore.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'react-redux';
+import books from './books/books';
+import categories from './categories/categories';
+
+const root = combineReducers({ books, categories });
+
+export default root;
```

Comment:
- [ ] Good job on setting up the Redux but there is a little issue, In your store configuration file, you are supposed to create a store so you can access your reducers and states in your application. Please create a store in this file.
  - You can use this code `
const store = createStore(
  root, applyMiddleware(logger, thunk),
);
`  And import required Redux methods and export store instead of root reducer.